### ISSUE TITLE
Lowest Points contains duplicate values

### ISSUE THREAD

**Issue by adgiles** [Esri/military-tools-geoprocessing-toolbox#242]
Running the lowest points tool in ArcMap, it seems that the results contain a point for every pixel in the lowest area, that are close together and of similar value, using the sample elevation in the testdata folder you get the following results:

![untitled](https://user-images.githubusercontent.com/3740334/27903231-722e3a54-6230-11e7-87f6-0fd05d8db2c1.png)

In this case the lowest points returned 2046 points as a grid around the lowest elevation values:

![untitled1](https://user-images.githubusercontent.com/3740334/27903238-7b931ba0-6230-11e7-90c7-18e3d59683b7.png)

I think the tool would be better if neighbouring pixels of the same values are converted to polygons and the center point of each polygon returned.

**Comment by adgiles** [comment]
Fix applied in PR #243, if change accepted please can somebody verify

**Comment by mfunk** [comment]
@adgiles That is correct. If your selection area includes a lake (which should be a low point) then the surface of the lake should have the same lowest values. w.r.t. spacing the number and arrangements shows the area of the lake. Reducing to a single point obscures this (a reason why it's called Lowest Points, and not Lowest Point)

**Comment by adgiles** [comment]
ok if that is the way the tool is designed and expected to work then PR #243 can be ignored

**Comment by mfunk** [comment]
@adgiles not ready to kill the creativity yet. So lets use this as a chance to think about improving this guy. So if a multiple point output doesn't work for flat, low areas... should we think about alternatives, like making an optional output low area polys?

**Comment by lfunkhouser** [comment]
FYI @topowright @ACueva @dfoll
